tests/solvers/test_sundials.py

```diff
@@ -164,6 +164,24 @@ def test_event_localizer(self):
         nose.tools.assert_almost_equal(y[-1][1],3.0)
         nose.tools.assert_almost_equal(y[-1][2],2.0)
     
+    @testattr(stddist = True)
+    def test_event_localizer_external(self):
+        exp_mod = Extended_Problem() #Create the problem
```

Comment:
I think we should add a docstring here explaining that we test that CVode works as expected when `external_event_detection` is set to `True`.

Reply:
Added comments